models/world_cricsheet/code/upload.py

```
@@ -0,0 +1,103 @@
+def upload_table(slug: str, rel_path: str, expected_rows: int) -> int:
+    path = OUTPUT_ROOT / rel_path
+    if not path.exists():
+        raise FileNotFoundError(f"Missing output path: {path}")
+
+    tb = bd.Table(dataset_id=DATASET_ID, table_id=slug)
+
+    # Delete stale GCS staging prefix (avoids BQ partition key conflicts)
+    st = bd.Storage(dataset_id=DATASET_ID, table_id=slug)
+    try:
+        st.delete_table(mode="staging", not_found_ok=True)
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)